src/main/java/com/thebois/views/IProjector.java

```diff
@@ -0,0 +1,20 @@
+package com.thebois.views;
+
+import com.badlogic.gdx.math.Vector2;
+
+/**
+ * Interface for a Projector.
+ */
+public interface IProjector {
+
+    /**
+     * Unprojects screen coordiantes to world coordinates.
+     *
+     * @param screenCordX X coordinate of the screen.
+     * @param screenCordY Y coordinate of the screen.
+     *
+     * @return Returns a Vector2 with the world coordinates.
+     */
+    Vector2 unProject(int screenCordX, int screenCordY);
```

Comment:
*unproject*

Reply:
Intellij's spelling checker complained but will change it and tell it to allow it.